src/libraries/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
+    <EventSourceSupport>true</EventSourceSupport>
```

Comment:
There are other similar tests, e.g. https://github.com/dotnet/runtime/search?q=LoggingTest.  Do other test projects need this as well?

Reply:
@radical do we have similar failures as https://github.com/dotnet/runtime/issues/56847 with `/p:EnableAggressiveTrimming=true`?

Reply:
At least this test have similar issues, https://github.com/dotnet/runtime/issues/52652.